plugins/pulp_rpm/plugins/importers/yum/upload.py

```diff
@@ -146,6 +146,11 @@ def _handle_erratum(repo, type_id, unit_key, metadata, file_path, conduit, confi
     steps are to save the metadata and optionally link the erratum to RPMs
     in the repository.
 
+    NOTE: For now errata is handled differently than other units. Uploaded erratum should not
+    overwrite the existing one if the latter exists, they should be merged. This is only because
+    of the way erratum is stored in the MongoDB and it is in our todo list to re-think how to do
```

Comment:
Please link to the place where that todo list is tracked? Like a bug in pulp.plan.io or similar.